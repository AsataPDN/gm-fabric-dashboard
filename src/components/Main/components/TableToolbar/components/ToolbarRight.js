import styled from "styled-components";

const ToolbarRight = styled.div`
  display: flex;
  flex: 1 1 33%;
  justify-content: flex-end;
`;

export default ToolbarRight;
