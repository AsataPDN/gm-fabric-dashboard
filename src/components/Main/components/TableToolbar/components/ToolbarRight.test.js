import React from "react";
import { shallow } from "enzyme";
import ToolbarRight from "./ToolbarRight";

describe("ToolbarRight", () => {
  it("matches snapshot", () => {
    const aToolbarRight = shallow(<ToolbarRight />);
    expect(aToolbarRight).toMatchSnapshot();
  });
});
