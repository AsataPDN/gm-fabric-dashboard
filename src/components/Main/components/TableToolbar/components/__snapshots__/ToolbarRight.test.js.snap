// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToolbarRight matches snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1 1 33%;
  -ms-flex: 1 1 33%;
  flex: 1 1 33%;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
}

<div
  className="c0"
/>
`;
